src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -142,17 +142,73 @@ public int getSampleRate() {
 	}
 
 	/**
+	 * @return True if the audio codec is 3GA.
+	 */
+	public boolean is3GA() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.THREEGA);
+	}
+
+	/**
+	 * @return True if the audio codec is AAC.
+	 */
+	public boolean isAAC() {
+		return getCodecA() != null && (getCodecA().equalsIgnoreCase(FormatConfiguration.AAC) || getCodecA().equalsIgnoreCase(FormatConfiguration.AAC_HE));
+	}
+
+	/**
+	 * @return True if the audio codec is AAC LC.
+	 */
+	public boolean isAACLC() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.AAC);
+	}
+
+	/**
 	 * @return True if the audio codec is AC-3.
 	 */
 	public boolean isAC3() {
 		return getCodecA() != null && (getCodecA().equalsIgnoreCase("ac3") || getCodecA().equalsIgnoreCase("a52") || getCodecA().equalsIgnoreCase("liba52"));
 	}
 
 	/**
-	 * @return True if the audio codec is TrueHD.
+	 * @return True if the audio codec is ADPCM.
 	 */
-	public boolean isTrueHD() {
-		return getCodecA() != null && getCodecA().equalsIgnoreCase("truehd");
+	public boolean isADPCM() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.ADPCM);
```

Comment:
@Sami32 That needs further confirmation. I thought I remembered something about that as well, but some of the current code still uses ```equalsIgnoreCase()``` so I didn't take the chance to remove them.